CONTRIBUTING.md

```diff
@@ -15,31 +17,84 @@ This document provides some overall guidelines and suggestions for how to get st
 2. Be sure to read our [code of conduct](http://www.codeforpdx.org/about/conduct)
 3. Say hi on our Slack Channel!
 
-### Guidelines
+### Using Git
+
+* First, be sure that your local repo is a clone of your fork and not of the main project repository (see the main README.md).
+* Define the `upstream` remote repository to be the main project repo on github:
+`git remote add upstream https://github.com/codeforpdx/recordexpungPDX`
+
+##### Creating a new feature branch
+
+* Navigate to your local master branch: `$ git checkout master`
+* Sync your local master branch with the upstream repository `$ git pull -r upstream master`
+* Create a new branch for the issue you would like to work on ie: `$ git checkout -b name-of-new-branch`
+
+At this point you should now be in your new branch and can begin working on the new feature or bug fix.
+
+##### Once you have changes that you would like to make available for others to see and use:
+* Run the backend unit tests and check that they all pass. In the main project directory, run `make dev_test`.
+* Add the changes: `$ git add .` There are many versions of this command, consult docs mentioned below for more details or run `$ git help add` to get help in the terminal.
+* Commit the changes: `$ git commit` This opens an editor to write your commit message. Consult articles about how to write a  good commit message; here is one by freeCodeCamp: https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/
+* Push the changes to *your* remote repository: `$ git push origin name-of-new-branch`
+* Open a Pull Request on the `codeforpdx/recordExpungPDX` repository referencing the issue that this PR addresses
+* Please do not push direcly to a branch on the shared repo.
```

Comment:
This line basically says the same thing as two lines above.

Reply:
it sure does!